### ISSUE TITLE
Someone have a acript that adds this to the healthbar?

### ISSUE THREAD

**Issue by Plotsh** [ShadowMario/FNF-PsychEngine#9137]
### What is your question?

as the discussions have been removed, im adking if someone have it, ik there was a discussion with it but rlly, needed to remove it?

i need someone who knows how to add these lines on the healthbar
![Screenshot_20220619-222916](https://user-images.githubusercontent.com/84821153/174510171-bb976ff5-f142-4967-ac8a-12e0ece93623.png)

**Comment by bruh1cat** [comment]
Just download Os engine it's basically psych but with improvements.

**Comment by Mastercopper2346** [comment]
What’s acript

**Comment by Plotsh** [comment]
> Just download Os engine it's basically psych but with improvements.

nah, i just want a script cuz im recreating cassette girl on psych

**Comment by silvashp** [comment]
this is the fucking psych enigne issues page, please stop things like "Use X engine,, its just psych but better!"
